forms-flow-api/src/formsflow_api/services/external/formio.py

```diff
@@ -0,0 +1,46 @@
+"""This exposes the Formio APIs."""
+import json
+from http import HTTPStatus
+
+import jwt
+import requests
+from flask import current_app
+
+from formsflow_api.exceptions import BusinessException
+from formsflow_api.utils import cache
+
+
+class FormioService:
+    """This class manages formio API calls."""
+
+    @classmethod
+    def get_access_token(cls):
+        """Method to get formio access token."""
+        headers = {"Content-Type": "application/json"}
+        url = f"{current_app.config.get('FORMIO_URL')}/user/login"
```

Comment:
can move the base URL to the class variable since it is accessed multiple times

Reply:
will update